### DISCUSSION TITLE
mnist_pretrained.npy

### DISCUSSION THREAD

**Discussion by Aniruddha120** [lava-nc/lava#527] [Q&A]
Dear developers,
Hope you are doing well. I was readin the docs of the LAVA framework, there an exemple of mnist model has been introduced. It requires mnist_pretrained.npy file. Can you help me to find this file or to download it? thank you!

**Comment by weidel-p** [comment]
This file is in the same folder as the mnist tutorial and should be automatically downloaded by Git. But if that's not working for you for any reason, here's the link: https://github.com/lava-nc/lava/raw/main/tutorials/end_to_end/mnist_pretrained.npy

I think you need to drop it in the same folder as the tutorial.

**Reply by Aniruddha120** [reply]
Hello, hope this message finds you well. I found your solution effective. Would please answer my bellow question also?
Q:  https://github.com/lava-nc/lava/discussions/526
